distribution/alliance/pom.xml

```diff
@@ -294,27 +294,27 @@
                         </configuration>
                     </execution>
                     <!--Include Documentation-->
-                    <execution>
-                        <id>unpack-alliance-documentation</id>
-                        <phase>generate-resources</phase>
-                        <goals>
-                            <goal>unpack</goal>
-                        </goals>
-                        <configuration>
-                            <artifactItems>
-                                <artifactItem>
-                                    <groupId>org.codice.alliance</groupId>
-                                    <artifactId>docs</artifactId>
-                                    <version>${project.version}</version>
-                                    <outputDirectory>${setup.folder}/documentation/alliance
-                                    </outputDirectory>
-                                    <classifier>docs-export</classifier>
-                                    <type>zip</type>
-                                </artifactItem>
-                            </artifactItems>
-                            <overWriteSnapshots>true</overWriteSnapshots>
-                        </configuration>
-                    </execution>
+                    <!--<execution>-->
+                        <!--<id>unpack-alliance-documentation</id>-->
+                        <!--<phase>generate-resources</phase>-->
+                        <!--<goals>-->
+                            <!--<goal>unpack</goal>-->
+                        <!--</goals>-->
```

Comment:
This was commented out by mistake. It is for unpacking the alliance documentation.